mypy/build.py

```diff
@@ -972,16 +977,20 @@ def get_cache_names(id: str, path: str, manager: BuildManager) -> Tuple[str, str
       pyversion: Python version (major, minor)
 
     Returns:
-      A tuple with the file names to be used for the meta JSON and the
-      data JSON, respectively.
+      A tuple with the file names to be used for the meta JSON, the
+      data JSON, and the deps JSON, respectively.
```

Comment:
Maybe call the deps JSON "fine-grained deps JSON" to make it clear that this is different from module dependencies?